docs/ARCHITECTURE.md

```
@@ -0,0 +1,277 @@
+- **アクセス解析なし**: Google Analytics等は入れない
+
+### シンプルな構成
+
+- ユーザーは初回設定後、基本的にシステムにアクセスしない
+- 通知メールから直接予約サイトへ遷移
+
+---
+
+## システム構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)